investigations/LOKI/loki_performance.md

```diff
@@ -24,12 +25,41 @@ The hardware used for these tests is as follows:
 - Intel(R) Core(TM) i9-9920X CPU @ 3.50GHz
 - Target Linux Platform (OpenMP is a hard requirement)
 
-## Live Consumption Tests
-### Event rates
-This document only explores performance at a target rate of 10<sup>7</sup>events/s as a worst case since day 1 instrument flux of 1MW, ramping up to 2MW, will not result in an incident flux beyond this for LOKI. See [ess-hardware-requirements-report-v1.pdf](report.pdf) produced by Simon Heybrock.
+## Outcomes
 
 ### Measurement Interpretation
 The ESS pulse rate for neutrons is expected to be **14Hz**. This will be the rate at which messaged will be produced in the Kafka system and therefore consumed by Mantid. Measurements are taken in this context. If Mantid is capable of processing events (writing events to in-memory data structures and performing basic reduction) at this rate at a target neutron flux, this indicates that Mantid can cope with the event rate.
+
+### Data ingest into Mantid
+
+Data consumption into Mantid using realistic SANS data generated using GEANT4 resulted in an average consumption rate of ~24Hz. This was performed at an event flux of ~1.4e<sup>7</sup> events per message. This clearly shows that Mantid can comfortably deal with the ingest of data at these high rates.
+
+##### Scaling
+There was previous work done in investigating how well the live streamer in Mantid scales and was presented in this document [LiveReductionInvestigation](https://github.com/DMSC-Instrument-Data/documents/blob/master/investigations/Live%20Reduction/LiveReductionInvestigation.md). Scaling for LOKI ingesting the simulated data showed the following trend:
+
+`#` of threads|Message Consumption Rate (Hz)
+---|---
+1|8
+4|26
+8|33
+16|52
+24 (Hyperthreading)|21
+
+Note that at 24 threads it is assumed the host system is oversubscribed due to running Mantid, Kafka broker and NeXus publisher simultaneously as well as hyperthreading.
```

Comment:
I might have not been overly clear, my point regarding HyperThreading was that loading all logical threads with a relatively constant workload is typically worse than loading one logical thread per physical core (due to context switching, etc.).

HyperThreading was always enabled, but at 24 logical threads these effects could have started to become visible. Although the sorting and insertion are fairly quick to execute so still think thread oversubscription was the issue.

Reply:
Feel free to make the relevant change to the document